### DISCUSSION TITLE
Upgrading chocolatey

### DISCUSSION THREAD

**Discussion by loctran016** [chocolatey/home#260] [Q&A] [answered]
So my current version of chocolatey is 1.1.0 and Window 11 and I want to upgrade it (`choco upgrade chocolatey` in admin cmd) but it output as follow:

```bash
Chocolatey v1.1.0
Upgrading the following packages:
chocolatey
By upgrading, you accept licenses for the packages.
chocolatey v2.0.0 is the latest version available based on your source(s).

Chocolatey upgraded 0/1 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```
Here is the log
[chocolatey.log](https://github.com/chocolatey/home/files/11648424/chocolatey.log)

**Comment by gep13** [comment]
@ItsBioDude can you please try first upgrading to 1.4.0 of Chocolatey CLI.

This can be done with `choco upgrade chocolatey --version 1.4.0`.

**Reply by pauby** [reply]
This is called out in our [upgrade guide](https://docs.chocolatey.org/en-us/guides/upgrading-to-chocolatey-v2-v6).

**Reply by loctran016** [reply]
Should I force it to update to 1.4.0 or not?
![image](https://github.com/chocolatey/home/assets/64778285/28366a61-300a-41e8-b84a-56902c42ed60)

**Reply by pauby** [reply]
Can you please read the upgrading guide I linked to? It answers that question, and many others.

**Reply by loctran016** [reply]
I read it careful but really I doesn't found the answer (the page is all about installing and about upgrading only `choco upgrade chocolatey` is about upgrading). Hmm but I try this and problem solved: `choco upgrade chocolatey --version 2.0.0 --force`. Thanks for help me anyway

**Reply by pauby** [reply, marked as answer]
[This section](https://docs.chocolatey.org/en-us/guides/upgrading-to-chocolatey-v2-v6#latest-1.x-and-5.x-stable-chocolatey-products) answers your question:

> To upgrade, you need to ensure you are at the latest stable version of the Chocolatey products that are installed on your computer. These are:
> 
>    Chocolatey CLI 1.4.0
> (others removed for brevity as they are not relevant to this question)
